js/src/forum/components/Search.js

```diff
@@ -70,7 +66,7 @@ export default class Search extends Component {
 
     // Initialize search input value in the view rather than the constructor so
     // that we have access to app.current.
-    if (typeof this.value() === 'undefined') {
+    if (!this.value()) {
```

Comment:
The comment above may no longer be correct?

Reply:
uh nope, not at all.